server/server.py

```diff
@@ -386,6 +408,29 @@ def do_GET(self):
                     message = json.dumps({"Error": "Error sending command to device.", "id": id})
             elif id != "":
                 message = json.dumps({"Error": "Device ID not found.", "id": id})            
+        elif self.path.startswith('/delayoff/'):
+            id = self.path.split('/delayoff/')[1]
+            sw = 1
+            delay = 10
+            if "/" in id:
+                try:
+                    (id, sw, delay) = id.split("/")
+                except:
+                    id = ""
+                    message = json.dumps({"Error": "Invalid syntax in delayoff command.", "url": self.path})
+            if id in deviceslist:
+                try:
+                    d = tinytuya.OutletDevice(id, deviceslist[id]["ip"], deviceslist[id]["key"])
+                    d.set_version(float(deviceslist[id]["version"]))
+
+                    thread = threading.Thread(target = delayoff, args = (d, sw, delay))
```

Comment:
Any particular reason you went with threading.Thread()+sleep() instead of threading.Timer()?